versioned_docs/version-23.10/releases/centreon-commercial-extensions.mdx

```diff
@@ -0,0 +1,124 @@
+---
+id: centreon-commercial-extensions
+title: Commercial Extensions
+---
+
+import ExpandCollapseAll from '@site/src/components/ExpandCollapseAll';
+
+## Introduction
+
+In this chapter, you can find all the changelogs concerning **Centreon Commercial Extensions**.
+
+> It is very important when you update your system to refer to this section in order to learn about behavior changes or
+> major changes that have been made on this version. This will inform you about the impact of installing these
+> versions on the features you use or the specific developments that you have built on your platform (modules,
+> widgets, plugins).
+
+If you have feature requests or want to report a bug, please contact support.
+
+<ExpandCollapseAll />
+
+## Centreon MAP
+
+### 23.10.0
+
+Release date: `October 27, 2023`
+
+<details open>
+  <summary>Enhancements</summary>
+
+</details>
+
+<details>
+  <summary>Compatibility</summary>
+
+- Compatibility with other 23.10 components.
+
+</details>
+
+## Centreon BAM
+
+### 23.10.0
+
+Release date: `October 27, 2023`
+
+<details open>
+  <summary>Enhancements</summary>
+
```

Comment:
```suggestion
- The output message for business activities has been modified to give more understandable information about the reason for the BA's status. This will particularly affect the clarity of the notifications.

```